packages/vscode-extension/src/handlers.ts

```diff
@@ -460,51 +460,10 @@ export async function openFolder(
   args?: any[]
 ) {
   await updateAutoOpenGlobalKey(showLocalDebugMessage, showLocalPreviewMessage, folderPath, args);
-  if (!TreatmentVariableValue.openFolderInNewWindow) {
-    await ExtTelemetry.dispose();
-    // after calling dispose(), let render process to wait for a while instead of directly call "open folder"
-    // otherwise, the flush operation in dispose() will be interrupted due to shut down the render process.
-    setTimeout(() => {
-      commands.executeCommand("vscode.openFolder", folderPath);
-    }, 2000);
-  } else {
-    const selection = await Promise.race([
-      new Promise<Result<string | undefined, FxError>>((resolve, reject) => {
-        setTimeout(resolve, 10000, err("timeout"));
-      }),
-      VS_CODE_UI.showMessage(
-        "info",
-        localize("teamstoolkit.handlers.openProject.title"),
-        false,
-        localize("teamstoolkit.handlers.openInNewWindow"),
-        localize("teamstoolkit.handlers.openInCurrentWindow")
-      ),
-    ]);
-    if (selection.isOk()) {
-      const openInNewWindow = selection.value === localize("teamstoolkit.handlers.openInNewWindow");
-      ExtTelemetry.sendTelemetryEvent(TelemetryEvent.OpenNewProject, {
-        [TelemetryProperty.VscWindow]: openInNewWindow
-          ? VSCodeWindowChoice.NewWindow
-          : VSCodeWindowChoice.CurrentWindow,
-      });
-      if (openInNewWindow) {
-        commands.executeCommand("vscode.openFolder", folderPath, true);
-      } else {
-        await ExtTelemetry.dispose();
-        // after calling dispose(), let render process to wait for a while instead of directly call "open folder"
-        // otherwise, the flush operation in dispose() will be interrupted due to shut down the render process.
-        setTimeout(() => {
-          commands.executeCommand("vscode.openFolder", folderPath);
-        }, 2000);
-      }
-    } else {
-      // timeout
-      ExtTelemetry.sendTelemetryEvent(TelemetryEvent.OpenNewProject, {
-        [TelemetryProperty.VscWindow]: VSCodeWindowChoice.NewWindowByDefault,
-      });
-      commands.executeCommand("vscode.openFolder", folderPath, true);
-    }
-  }
+  ExtTelemetry.sendTelemetryEvent(TelemetryEvent.OpenNewProject, {
```

Comment:
Do we still need the 'TelemetryProperty.VscWindow' property while other entries are already removed?

Reply:
Yes, I'm keeping this one in case we add other entries in the future.

Reply:
Please be aware that event with this property doesn't mean the same for versions before&after this change when querying data.